main.tf

```diff
@@ -73,3 +73,66 @@ module "cbr_rule" {
     tags = var.cbr_rules[count.index].tags
   }]
 }
+
+########################################################################################################################
+# SCC Workload Protection Trusted Profile
+########################################################################################################################
+
+# Create Trusted profile for SCC Workload Protection instance
+module "trusted_profile_scc_wp" {
+  count                       = var.cspm_enabled ? 1 : 0
+  source                      = "terraform-ibm-modules/trusted-profile/ibm"
+  version                     = "2.1.1"
+  trusted_profile_name        = var.scc_workload_protection_trusted_profile_name
+  trusted_profile_description = "Trusted Profile for SCC workload Protection instance ${ibm_resource_instance.scc_wp.guid} with required access for configuration aggregator."
+
+  trusted_profile_identity = {
+    identifier    = ibm_resource_instance.scc_wp.crn
+    identity_type = "crn"
+  }
+
+  trusted_profile_policies = [
+    {
```

Comment:
after bumping to 3.0.0, you will need to add `unique_identifier` attribute. It can be any value since we are not upgrading from old version of module. Suggest to go with `unique_identifier = "app-config-access"`